packages/lib/bookings/findQualifiedHosts.ts

```diff
@@ -109,16 +109,8 @@ export const findQualifiedHosts = async <
   );
 
   if (hostsAfterRescheduleWithSameRoundRobinHost.length === 1) {
-    // push all disqualified hosts into fallback, reschedule with same user is recoverable
-    const fallbackHosts = getFallbackHosts(
-      fixedHosts,
-      roundRobinHosts,
-      hostsAfterRescheduleWithSameRoundRobinHost,
-      "NotSameRoundRobinHost"
-    );
     return {
       qualifiedHosts: [...fixedHosts, ...hostsAfterRescheduleWithSameRoundRobinHost],
-      fallbackHosts,
```

Comment:
Currently, we don't have a fallback when same RR host has no availability for two week. I would keep it like that for now